src/timeseriesflattener/feature_spec_objects.py

```diff
@@ -714,6 +716,16 @@ def is_dichotomous(self) -> bool:
 
         return len(self.values_df[col_name].unique()) <= 2  # type: ignore
 
+    def is_incident(self):
+        """Check if the outcome is incident, i.e. contains unique IDs."""
+        if self.incident:
+            if self.values_df[self.entity_id_col_name].nunique() != len(
+                self.values_df,
+            ):
+                log.warning(
+                    "Incident outcomes must have unique IDs. You have set incident=[True], however, the keys in your ID column are not unique. Consider setting incident=[False].",
+                )
```

Comment:
Tænker det vil være godt reelt at raise en `ValueError` 👍

Reply:
I stedet eller også?

Reply:
I stedet eller også?

Reply:
I stedet for, men med samme besked 👍

Reply:
I stedet eller også?